src/p4p/rpc.py

```diff
@@ -264,13 +264,14 @@ def _wrapMethod(K, V):
     if S.varargs is not None or keywords is not None:
         raise TypeError("vararg not supported for proxy method %s" % K)
 
-    if len(S.args) != len(S.defaults):
+    args, defaults = ([] if v is None else v for v in (S.args, S.defaults))
```

Comment:
Do you mean for this line to do something like the following?

```py
args, defaults = args or [], defaults or []
```

Reply:
Yes, but I generally don't use the `or` syntax to handle `None` because it also brakes on falsely values.

Reply:
For reference, I understand the concern about loose type comparison rules.  In this instance, `S` comes from the builtin `inspect` module, which I think can be relied on not to change radically.  So I think the `or []` is a reasonable abbreviation.